kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -81,44 +81,71 @@
       </KGrid>
 
       <h2>{{ $tr('chooseExercises') }}</h2>
+      <div v-if="!showChannels">
+        <br>
```

Comment:
generally it's preferable to use CSS rather than HTML for layout spacing (e.g. add a top margin)